src/AbstractRoboFile.php

```diff
@@ -1315,6 +1317,30 @@ protected function getLocalSettings($app = null, $timestamp = null)
         return $this->tokenReplace($this->getConfig()->get('local'), $replacements) + $defaults;
     }
 
+    protected function getTimeoutSetting($setting = null)
```

Comment:
Shouldn't default settings be in the property files?

Reply:
They are:
`$timeoutSettings = $this->getConfig()->get('timeouts', []) + ...`